clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -364,7 +365,8 @@ public enum Errors {
     INCONSISTENT_TOPIC_ID(103, "The log's topic ID did not match the topic ID in the request", InconsistentTopicIdException::new),
     INCONSISTENT_CLUSTER_ID(104, "The clusterId in the request does not match that found on the server", InconsistentClusterIdException::new),
     TRANSACTIONAL_ID_NOT_FOUND(105, "The transactionalId could not be found", TransactionalIdNotFoundException::new),
-    FETCH_SESSION_TOPIC_ID_ERROR(106, "The fetch session encountered inconsistent topic ID usage", FetchSessionTopicIdException::new);
+    FETCH_SESSION_TOPIC_ID_ERROR(106, "The fetch session encountered inconsistent topic ID usage", FetchSessionTopicIdException::new),
+    USER_AUTHORIATION_FAILED(107, "User authorization failed", UserAuthorizationException::new);
```

Comment:
minor typo: `USER_AUTHORIATION_FAILED` => `USER_AUTHORIAZATION_FAILED`

Reply:
Also we are not using this error. Can we remove this. We can just `DELEGATION_TOKEN_AUTHORIZATION_FAILED` error for all token related authorisation issues.